src/ShopPaymentTransactionBundle/objects/TCMSTableEditor/TPkgShopPaymentTransaction_TCMSTableEditorShopOrder.class.php

```diff
@@ -153,6 +153,14 @@ public function pkgShopPaymentTransaction_PartialDebit()
 
         $oTransactionManager = new TPkgShopPaymentTransactionManager($oOrder);
 
+        if (0.00 === $transactionValue) {
```

Comment:
Equality checks on floats will fail sometimes - we should check for something like `$transactionValue < 0.0001`

Reply:
changed